Libraries/src/Amazon.Lambda.AspNetCoreServer/APIGatewayProxyFunction.cs

```diff
@@ -37,6 +39,16 @@ public abstract class APIGatewayProxyFunction
         public bool EnableResponseLogging { get; set; }
 
         /// <summary>
+        /// If true then all response content will be Base64-encoded before being returned.
+        /// </summary>
+        public bool EnableBinaryResponseContent { get; set; }
+
+        /// <summary>
+        /// An optional set of Content-Types that will trigger encoding the response content in Base64.
+        /// </summary>
+        public IEnumerable<string> BinaryResponseContentTypes { get; set; }
```

Comment:
Just occurred to me there is one more dependency to consider with whatever interface changes are made to support this, namely that you also have to update (and keep in sync) the [Binary Support](http://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-payload-encodings-configure-with-console.html) settings of the API Gateway by enumerating the binary content-types that your app will support.